main/src/cgeo/geocaching/ui/CacheListAdapter.java

```diff
@@ -426,8 +430,8 @@ public View getView(final int position, final View rowView, final ViewGroup pare
         else {
             holder.text.setText(cache.getName());
         }
-        holder.text.setCompoundDrawablesWithIntrinsicBounds(MapUtils.getCacheMarker(res, cache, cacheListType), null, null, null);
-
+        holder.cacheListType = cacheListType;
+        updateViewHolder(holder, cache, res);
```

Comment:
cacheListType is needed in updateViewHolder. This is now called from  main/src/cgeo/geocaching/ui/LoggingUI.java where I can get the holder but not the CacheListType. Therefore cacheListType needs to be stored in the holder.